### PR TITLE
treewide: s/Yuibco/Yubico/

### PR DESCRIPTION
**Pull request by stintel**
Signed-off-by: Stijn Tintel <stijn@linux-ipv6.be>

Maintainer: @stuwilkins
Compile tested: n/a
Run tested: n/a

Fix typo

### CODE REVIEW

**Review by BKPepe — APPROVED** [review]
This thing might require bumping PKG_RELEASE, otherwise :+1:

**Comment by stintel** [comment]
> This thing might require bumping PKG_RELEASE, otherwise +1

Do we really want that for fixing a typo?

**Comment by BKPepe** [comment]
It's up to you. It's questionable. I am fine with or without bumping `PKG_RELEASE`.

**Comment by stintel** [comment]
> It's up to you. It's questionable. I am fine with or without bumping `PKG_RELEASE`.

Ok, I'll wait for another opinion ;-)

**Comment by champtar** [comment]
The rule is simple IMO, does the package checksum change ?

**Comment by stintel** [comment]
Patches need refreshing, I don't even use those packages, so I'm closing this PR.

**Comment by BKPepe** [comment]
Just cc: @aparcar on this one as he takes care of CI. ;-) Maybe he can do it opt-in.

**Comment by aparcar** [comment]
> Just cc: @aparcar on this one as he takes care of CI. ;-) Maybe he can do it opt-in.

Can you clarify this? You want to make the CI optional?

**Comment by BKPepe** [comment]
Sure, I can clarify on this one.

CI currently fails about of:
>  Dirty patches detected, please refresh and review the diff 

See: https://github.com/openwrt/packages/runs/1941872056#step:5:3512

Let's imagine a situation.  Here you have a contributor who wants to fix a typo. But he rather gives us because doing refreshing patches might as it could bring its own issues, and most importantly, it is not related to his commit anyhow. 

So,  it should be a separate check, which would be optional. I am not sure how did you come to an idea that CI should be opt-in, though.

Also https://freenode.irclog.whitequark.org/openwrt-devel/2021-02-20#29190635;

**Comment by stintel** [comment]
Apparently it's impossible to reopen a PR once the branch has been deleted, even after restoring it to its original state. Top stuff, Github. See #14814

**Comment by aparcar** [comment]
> Apparently it's impossible to reopen a PR once the branch has been deleted, even after restoring it to its original state. Top stuff, Github. See #14814

Yea I had the same issue, not sure why GitHub behaves that way. I'm guessing to lower the PR tracking load.

@BKPepe It's always possible to merge even if the CI doesn't like it. If the changes are trivial you can merge even if the CI hates it.
 
> I am not sure how did you come to an idea that CI should be opt-in, 

Wait what? Didn't you just write one comment before `Maybe he can do it opt-in`?

Regarding the IRC log, I'm assuming s/gitlab/github/ and I don't see how it's blocking, even if CI fails I can click the button, it just gives a warning:

image

**Comment by neheb** [comment]
@BKPepe The barrier for making a PR is already so great that adding a make package/x/refresh requirement does not add much.

I plan to merge @lipnitsk 's PR in order to reduce problems like these for new contributors. https://github.com/openwrt/packages/pull/14646

**Comment by stintel** [comment]
Actually if we enforce that kind of things via CR we should just fix the entire tree ourselves, so that people don't get demotivated like I did earlier today.

**Comment by aparcar** [comment]
I ran a script over all packages and only found a single package with dirty patches, see https://github.com/openwrt/packages/pull/14824

**Comment by BKPepe** [comment]
> Wait what? Didn't you just write one comment before Maybe he can do it opt-in?

I think we don't understand each other. You merged `Detect dirty patches,` and in my point of you, we should do it optional or at least have the possibility to skip such test as CI fails and doesn't proceed with building step and so on. It could be done as a separate job/check, where it says that there is something wrong happening with patches, which he didn't touch. Otherwise, if all tests are failing, it can demotivate users, and it does not encourage users to contribute anymore. As it happened in this PR.

I want to avoid a situation where we are putting more and more bricks into walls against contributors, as said here https://freenode.irclog.whitequark.org/openwrt-devel/2021-02-21#29195725; because even at this time, there is a lack of contributors.

TL;DR: @stintel is right, and it was also done in https://github.com/openwrt/packages/pull/14646
I like changes or improvements over CI, but not all of them should be mandatory.
